app/faceanalysis/models/models.py

```diff
@@ -9,7 +8,8 @@
                         Integer, DateTime, UniqueConstraint)
 from itsdangerous import (TimedJSONWebSignatureSerializer
                           as Serializer, BadSignature, SignatureExpired)
-from .database_manager import DatabaseManager
+from .database_manager import get_database_manager
+from ..settings import TOKEN_EXPIRATION, TOKEN_SECRET_KEY
```

Comment:
In previous PRs, ..imports were replaced with module.module.module. Which should we choose for consistency?